tests/cloud-control-server/README.md

```diff
@@ -0,0 +1,22 @@
+## Databend Cloud Control grpc Server Tests
+
+
+### How to run
```

Comment:
https://github.com/datafuselabs/databend/pull/13482

Please use the snake name:`cloud_control_server`.